test/new_relic/agent/instrumentation/rails/active_job_subscriber.rb

```diff
@@ -31,12 +31,12 @@ class ActiveJobSubscriberTest < Minitest::Test
     SUBSCRIBER = NewRelic::Agent::Instrumentation::ActiveJobSubscriber.new
 
     def test_segment_naming_with_unknown_method
-      assert_equal 'Ruby/ActiveJob/default/Unknown',
+      assert_equal 'Ruby/ActiveJob/TestJob/default/Unknown',
```

Comment:
Do we have tests somewhere with more deeply nested names? something like `MyApp::Jobs::EmailNotificationJob`? The code behaves correctly with names like this, but I'm worried about potential regressions.

Reply:
I added a test that is more nested, just stealing from your idea `MyApp::Jobs::EmailNotificationJob`. There is now another job class and a test to grab the name!